yorkie/rpc/broadcast.go

```diff
@@ -0,0 +1,35 @@
+package rpc
+
+import (
+	"context"
+
+	"github.com/yorkie-team/yorkie/api"
+	"github.com/yorkie-team/yorkie/api/converter"
+	"github.com/yorkie-team/yorkie/internal/log"
+	"github.com/yorkie-team/yorkie/pkg/document/time"
+)
+
+// Publish publishes the given event to the given Topic.
+func (s *Server) Publish(_ context.Context, request *api.PublishRequest) (*api.PublishResponse, error) {
+	actorID, err := time.ActorIDFromBytes(request.PublisherId)
+	if err != nil {
+		log.Logger.Fatal(err)
```

Comment:
`Fatal` kills the server. Do you have any reason to kill me here?

Reply:
This seems to be wrong. I'll fix it.